app/views/self_service/firms/index.html.erb

```diff
@@ -1,14 +1,21 @@
 <%= render_breadcrumbs(breadcrumbs_root) %>
 
 <div class="l-self-service">
-  <h2><%= t('self_service.firms_index.firm_heading') %></h2>
+  <h1 class="t-page-title">
```

Comment:
Was this a deliberate change from h1 to h2? Does the document outline still match up?